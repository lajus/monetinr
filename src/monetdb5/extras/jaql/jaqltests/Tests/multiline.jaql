[
	{
		"key": "value",
		"otherkey": "othervalue"
	}
]
	-> transform $.key
;
