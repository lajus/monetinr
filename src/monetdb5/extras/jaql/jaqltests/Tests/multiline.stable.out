stdout of test 'multiline` in directory 'monetdb5/extras/jaql/jaqltests` itself:


# 21:17:41 >  
# 21:17:41 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "gdk_dbfarm=/net/sofia.ins.cwi.nl/export/scratch1/fabian/tmp/mtest-Oct2012-sofia.ins.cwi.nl/five/dbfarm" "--set" "mapi_open=true" "--set" "mapi_port=36224" "--set" "monet_prompt=" "--trace" "--forcemito" "--set" "mal_listing=2" "--dbname=mTests_extras_jaql_jaqltests" "--set" "mal_listing=0"
# 21:17:41 >  

# MonetDB 5 server v11.13.0 "Oct2012-b3df9ec1a3e0+"
# Serving database 'mTests_extras_jaql_jaqltests', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 15.629 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2013 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://sofia.ins.cwi.nl:36224/
# MonetDB/GIS module loaded
# MonetDB/JAQL module loaded
# MonetDB/SQL module loaded
# MonetDB/DataCell loaded

Ready.

# 21:17:42 >  
# 21:17:42 >  "mclient" "-ljaql" "-ftest" "-Eutf-8" "-i" "-e" "--host=sofia" "--port=36224"
# 21:17:42 >  

#[
#	{
#		"key": "value",
#		"otherkey": "othervalue"
#	}
#]
#	-> transform $.key
#;
% .json # table_name
% json # name
% clob # type
% 11 # length
[ "value" ]

# 21:17:42 >  
# 21:17:42 >  "Done."
# 21:17:42 >  

